connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -427,4 +435,62 @@ public void ensureValid(String name, Object value) {
         }
     }
 
+    private static class ResponseHttpHeadersValidator implements ConfigDef.Validator {
+        @Override
+        public void ensureValid(String name, Object value) {
+            String strValue = (String) value;
+            if (strValue.isEmpty()) {
+                return;
+            }
```

Comment:
Should probably add this as the first bit in this method:
```suggestion
            String strValue = (String) value;
            if (value == null || strValue.trim().isEmpty()) {
                return;
            }
```